code/src/Styles/Form.css

```diff
@@ -0,0 +1,162 @@
+h1 {
+	font-size: 26px;
+	text-align: center;
+	margin-bottom: 20px;
+}
+h2 {
+	font-size: 22px;
+	margin-bottom: 20px;
+}
+
+h3 {
+	font-size: 20px;
+	margin: 20px 0;
+}
+
+h4 {
+	font-size: 24px;
+	font-weight: bold;
+	margin-bottom: 20px;
+	text-align: center;
+}
+
+.content-container {
+	display: flex;
+	flex-direction: column;
+}
+
+form {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	justify-content: space-evenly;
+	width: 100%;
+	height: 80%;
+}
+
+.form-wrapper {
+	background-image: linear-gradient(
+			to bottom,
+			rgba(0, 0, 0, 0.4),
+			rgba(0, 0, 0, 0.3)
+		),
+		url('../Images/season-unsplash.jpg');
+	background-size: cover;
+	background-position-x: center;
+	background-position-y: 45%;
+	height: 100vh;
+	width: 100vw;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+.introduction {
+	padding: 0 10px;
+	height: 100vh;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.navigation-start,
+.navigation-reset {
+	display: flex;
+	justify-content: center;
+}
+
+.question-container,
+.reset-wrapper {
+	padding: 0 10px;
+	width: 100%;
+	color: #fdf7ef;
+	display: flex;
+	flex-direction: column;
+	height: 70%;
+	justify-content: center;
+}
+
+/*Styling for text input*/
+.input-text-container {
+	border: none;
+	border-radius: 5px;
+	opacity: 0.5;
+	height: 30px;
+	width: 100%;
+	font-size: 16px;
+	padding-left: 10px;
+}
+
+/*Styling for select drop down*/
+select {
+	height: 30px;
+	width: 100%;
+	font-size: 16px;
+	border: none;
+	border-radius: 5px;
+}
+
+/*Styling for input email*/
+.input-email-container {
+	border: none;
+	border-radius: 5px;
+	opacity: 0.5;
+	height: 30px;
+	width: 100%;
+	font-size: 16px;
+	padding-left: 10px;
+}
+
+/*Styling for summary*/
+.question-container-summary {
+	padding: 0 10px;
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	width: 100%;
+	justify-content: space-evenly;
+}
+
+.summary-container {
+	color: #fdf7ef;
+	line-height: 30px;
+	background: black;
```

Comment:
Consider being consistent in using hex or plain text for the colors. Here you could use #000 instead since you are using HEX in a lot of other places in the code.

Reply:
True, I have updated this =)